firedrake/fml/form_manipulation_language.py

```diff
@@ -167,8 +168,8 @@ class LabelledForm(object):
     """
     A form, broken down into terms that pair individual forms with labels.
 
-    The `LabelledForm` object holds a list of terms, which pair
-    :class:`ufl.Form` objects with :class:`.Label`\\ s. The :meth:`label_map`
+    The LabelledForm object holds a list of terms, which pair
+    ufl.Form objects with .Label\\ s. The label_map
```

Comment:
Question: when you build the docs, does `.Label\\ s` link to the class correctly? Do you need the `.` before?